ludwig/utils/automl/data_source.py

```diff
@@ -1,9 +1,13 @@
 from abc import ABC, abstractmethod
 from typing import List, Tuple
 
+import dask.dataframe as dd
```

Comment:
This dask import should be ok since users using Ludwig automl are expected to `pip install ludwig[distributed]`. Is that right?

Reply:
Yeah, the automl library requires Ray and Dask to work.